### PR TITLE
Bugfix/ui improvements

### PR DESCRIPTION
**Pull request by michaeldera**
made general improvements to the ui and added a mobile skin for smaller devices.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <meta property="og:title" content="EDLIZ  | Essential Drugs and Medicines List  for Zimbabwe" />
     <meta property="og:description" content="This 7th essential medicines list and standard treatment guidelines for the most common health conditions in Zimbabwe has been endorsed by the National Medicine & Therapeutics Policy Advisory Committee [NMTPAC]." />
     <meta property="og:image" content="https://edliz.azurewebsites.net/img/pills.jpg" /> 
-    <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Serif|Muli:300,400,600" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Serif|Muli:600" rel="stylesheet">
```
Comment (Noeza) [inline public/index.html]: Is there a reason why you decided to remove those font weights, are the fonts still working as expected after removing them?

File: src/Components/MainMenu/MainMenu.js [hunk 2]
```diff
@@ -1,17 +1,32 @@
...
+            visible: !state.visible
+        }));
+    }
+
     render () {
         return (
-            <div className= "header">
-                 <div className="menu-list">
+            <header className={this.state.visible? "nav-on": "nav-off"}>  
+            <Media query="(max-width: 780px)" render={() => <HamburgerButton action={this.toggleNavigation}/>} />
```
Comment (Noeza) [inline src/Components/MainMenu/MainMenu.js]: Please set the outline:none for the Hamburger Button

File: src/Views/HamburgerButton/HamburgerButton.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import bars from './menu.svg'
+
+export default class HamburgerButton extends React.Component {
+    render(){            
+        return (
+            <button style={{position:"fixed", top: 0, left:0, zIndex: 99999, border: "none", padding: 16, margin:0, boxSizing: "border-box", background: "#fff", width: 50, height: 50, top: 0, left: 0}} onClick={this.props.action} className="btn-toggle-nav">
```
Comment (Noeza) [inline src/Views/HamburgerButton/HamburgerButton.js]: For the sake of consistency, its best we keep all css in the css files. It makes code readability easier.

**Comment by Noeza** [comment]
Just a few questions on the choices you made:
1. Why do yu think its best to capitalize this title
image
2. The reading text you set it to #666666 a shade of gray. It may just be me, but get a second opinion from someone about how they feel about it, i think the color contrast for readablity is a bit strenuous  to the eyes as #666666 is a bit lighter. Might i suggest tryin out #444444 and see what you think.
3. In the reading compoment where you have your chapters, i believe its essential that a link to `About this Project, instead of having to go home first and then.....
4. Be careful not to use too much blue especially as a background color, and we dont want to clutter the web with too many different color and fonts, the simpler it is the better. 
5. I think you can do better than this design lol. rethink it, its not aesthetically pleasing.
image

File: public/index.html (hunk 1 above)
Comment (michaeldera) [reply public/index.html]: Yes, if you compare this PR to the live version of the source you will see that there was no place were the other font-weights were being used. So I thought it just reduces performance... unless I missed that.

File: src/Components/MainMenu/MainMenu.js (hunk 2 above)
Comment (michaeldera) [reply src/Components/MainMenu/MainMenu.js]: Cool. Will add a commit to this effect.

File: src/Views/HamburgerButton/HamburgerButton.js (hunk 3 above)
Comment (michaeldera) [reply src/Views/HamburgerButton/HamburgerButton.js]: Agreed but perhaps we also need to come up with a naming convention... while css classes are imported at component level, the rule would applies if it matches another element in a separate component with the same identifier.

**Comment by Noeza** [comment]
menu
@michaeldera might i suggest something of this nature if you can pull it off. What do you think?

**Comment by michaeldera** [comment]
I can pull that off but it would not work well on mobile devices.

**Comment by Noeza** [comment]
> I can pull that off but it would not work well on mobile devices.

Sweet, no stress.

**Comment by michaeldera** [comment]
Done, with the exception of the Fonts thing, I figure it should be a separate branch.

**Comment by michaeldera** [comment]
@noeza have you had a chance to check this out?

**Comment by Noeza** [comment]
Will do just now

**Pull request merged**